deriv.com.conf

```
@@ -1,13 +1,16 @@
 server {
     listen 80;
     server_name  _;
+    #### Default Cache headers. This to prevent caching non-existent pages while releasing new versions. Reference -  https://www.gatsbyjs.com/docs/caching/  #######
```

Comment style is unusual to me.
```suggestion
    # By default not cache for most resources, recommended by Gatsby: https://www.gatsbyjs.com/docs/caching/ 
   # As well as being recommended by Gatsby, this ensures our releases take immediate effect
```